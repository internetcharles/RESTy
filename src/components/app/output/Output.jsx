import React from 'react';
import PropTypes from 'prop-types';

const Output = ({ results }) => (
  <figure>
    <span>{results}</span>  
  </figure>
);

Output.propTypes = {
  results: PropTypes.string.isRequired
};

export default Output;
