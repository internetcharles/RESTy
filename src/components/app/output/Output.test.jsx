import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Output from './Output';

describe('output component', () => {
  afterEach(() => cleanup());
  it('renders output', () => {
    const { asFragment } = render(<Output
      results="yo"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
