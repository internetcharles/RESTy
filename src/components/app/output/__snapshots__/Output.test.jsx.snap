// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output component renders output 1`] = `
<DocumentFragment>
  <figure>
    <span>
      yo
    </span>
  </figure>
</DocumentFragment>
`;
